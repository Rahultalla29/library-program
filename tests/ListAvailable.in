LIST AVAILABLE
ADD BOOK sample1.csv 111111
LIST AVAILABLE
ADD MEMBER Jeff Holmes
RENT 100000 111111
LIST AVAILABLE
RELINQUISH 100000 111111
LIST AVAILABLE LONG
EXIT
