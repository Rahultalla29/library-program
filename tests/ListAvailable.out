user: No books in system.

user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: To Kill a Mockingbird (Harper Lee)

user: Success.

user: Success.

user: No books available.

user: Success.

user: 111111: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

user: Ending Library process.
